### PR TITLE
redshift ssh tunnel

### PR DESCRIPTION
**Pull request by leoromanovsky**

### CODE REVIEW

File: docs/connecting-your-data/data-warehouses/connecting-to-redshift.md [hunk 1]
```diff
@@ -72,6 +72,23 @@ You'll want to gather the following connection details from Redshift:
...
+
+* tunnel host
+* username & password
+
+**SSH tunnel with public key**
+
+* tunnel host
+* username
+* public key
+
```
Comment (chasdevs) [inline docs/connecting-your-data/data-warehouses/connecting-to-redshift.md:91]: Should we add a screenshot or some actual example? It always makes me feel better when I'm reading docs and there's a specific example.

**Review by chasdevs — APPROVED** [review]

File: docs/connecting-your-data/data-warehouses/connecting-to-redshift.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Connecting to Redshift
 
-## 1. Whitelist Eppo IP Address
+## 1. Allowlist Eppo IP Address
```
Comment (schmit) [inline docs/connecting-your-data/data-warehouses/connecting-to-redshift.md:3]: Nit: I like this change but allowlist sounds more like a noun than a verb to me, making this sound a little odd, maybe just `Allow Eppo IP address`?

**Review by schmit — APPROVED** [review]
As a famous Leo likes to say: Good stuff!

File: docs/connecting-your-data/data-warehouses/connecting-to-redshift.md (hunk 2 above)
Comment (petzel) [reply docs/connecting-your-data/data-warehouses/connecting-to-redshift.md:3]: Could also be `Add Eppo IP addresses to Allowlist`

**Pull request merged**